app/scripts/models/reliers/oauth.js

```diff
@@ -30,6 +30,7 @@ define(function (require, exports, module) {
     client_id: Vat.clientId().required().renameTo('clientId'),
     code_challenge: Vat.codeChallenge().renameTo('codeChallenge'),
     code_challenge_method: Vat.codeChallengeMethod().renameTo('codeChallengeMethod'),
+    keys_jwk: Vat.keysJwk().renameTo('keysJwk'),
```

Comment:
I don't see this in a functional test, are those forthcoming?

Reply:
I filed a follow up issue here after this PR: https://github.com/mozilla/fxa-content-server/issues/5686